packages/toolkit/src/query/tests/buildSlice.test.ts

```diff
@@ -77,13 +75,26 @@ describe('buildSlice', () => {
             status: 'fulfilled',
           },
         },
-        subscriptions: {
-          'getUser(1)': expect.any(Object),
-        },
+        // Filled in a tick later
+        subscriptions: expect.any(Object),
       },
       auth: {
         token: '1234',
       },
+    }
+
+    expect(storeRef.store.getState()).toEqual(initialQueryState)
+
+    await delay(1)
+
+    expect(storeRef.store.getState()).toEqual({
+      ...initialQueryState,
+      api: {
+        ...initialQueryState.api,
+        subscriptions: {
+          'getUser(1)': expect.any(Object),
+        },
+      },
```

Comment:
why did you remove that part?

Reply:
Ah... good question. I think it was that the test seemed more focused on "was there _any_ state at all before, and did it all get cleared out by the reset action", rather than the specifics of exactly _which_ state fields existed, and I was getting tired of adding `delay()` calls to make things line up.